examples/document/document.css

```diff
@@ -5,18 +5,49 @@
   margin: 0.5in 0.67in 0.67in 0.67in;
 }
 
+@page :first {
+  @bottom-right-corner,
+  @bottom-left-corner {
+    content: unset;
+  }
+}
+
+@page :right {
+  @bottom-right-corner {
+    content: counter(page);
+  }
+}
+
+@page :left {
+  @bottom-left-corner {
+    content: counter(page);
+  }
+}
+
 html {
   font: 10.5pt/1.15 'Noto Serif', serif;
   text-align: justify;
 }
 
+p[data-split-to], li[data-split-to], figcaption[data-split-to], blockquote[data-split-to],
+.sectionbody, [data-split-original] {
+  text-align-last: left !important;
+}
+
 #cover {
   display: flex;
   flex-direction: column;
   justify-content: center;
   height: 100vh;
   page-break-after: always;
   align-items: flex-end;
+
+  counter-increment: page -1;
```

Comment:
Should we remove it for now (and/or add a comment with a link to the GitLab issue ?)